src/current/v24.3/create-virtual-cluster.md

```diff
@@ -49,7 +49,7 @@ Parameter | Description
 
 Option | Description
 -------+-------------
-`READ VIRTUAL CLUSTER` | 
+<span class="version-tag">New in v24.3:</span> `READ VIRTUAL CLUSTER` | Configure the PCR stream to allow reads from the standby cluster. For more details, refer to [Start a PCR stream with read from standby](#start-a-pcr-stream-with-read-from-standby).
```

Comment:
nit: could this description make it explicit that this option creates a new vc on the standby?

something like "create read only virtual cluster on the standby host cluster"?

Reply:
Good point, I _think_ I've taken care of this.